recipe/meta.yaml

```diff
@@ -1,5 +1,7 @@
 {% set version = "7.3.0b2" %}
 
+{% set python_min = python_min | default("3.9") %}
```

Comment:
Upstream has not dropped `3.8` yet: https://github.com/jupyter/notebook/blob/606190117a34f9488bbdf87d90feb219755a6153/pyproject.toml#L10

But I guess it's fine to do it here already?

Reply:
Right, but `conda-forge` _has_ already dropped the EOL python: https://github.com/conda-forge/cfep/blob/main/cfep-25.md